app/gui2/src/components/GraphEditor.vue

```diff
@@ -106,6 +137,10 @@ function moveNode(id: ExprId, delta: Vec2) {
         v-for="[id, node] in graphStore.nodes"
         :key="id"
         :node="node"
+        :main-module="mainModule"
+        :execution-context-id="executionContextId"
+        :language-server="projectStore.lsRpcConnection"
+        :data-server="projectStore.dataConnection"
```

Comment:
i was assuming `v-bind` props were supposed to stay as `kebab-case` - can be changed 

re: the low level part of managing the execution context and visualization messages - yeah, this will probably be addressed in adam's PR, i'll probably wait for that to land before merging that into this PR.

i'll see what i can do before that gets merged in though